images/linux/scripts/installers/kotlin.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -e
+################################################################################
+##  File:  kotlin.sh
+##  Desc:  Installs Kotlin
+################################################################################
+
+source $HELPER_SCRIPTS/install.sh
+
+KOTLIN_ROOT="/usr/local"
+
+URL=$(curl -s https://api.github.com/repos/JetBrains/kotlin/releases/latest | jq -r '.assets[].browser_download_url | select(contains("kotlin-compiler"))')
+download_with_retries $URL "/tmp"
+
+unzip -qq /tmp/kotlin-compiler*.zip -d $KOTLIN_ROOT
+rm $KOTLIN_ROOT/kotlinc/bin/*.bat
+ln -sf $KOTLIN_ROOT/kotlinc/bin/* /usr/local/bin/
```

Comment:
let's symlink to `/usr/bin` instead of `/usr/local/bin` so that anyone will be able to override our installation with their own